### ISSUE TITLE
New database -- ASNHC:Bird

### ISSUE THREAD

**Issue by diannakrejsa** [ArctosDB/data-migration#168]
We're ready to start working on migration of other databases (Specify-to-Arctos mammal migration is done, woohoo!), and we've decided to go with the birds collection next. 

Teresa, can you add a database to our set of ASNHC-managed databases, ASNHC:Bird? Is this something that I should be able to do or do we contact you all for set up of a new collection?

**Comment by campmlc** [comment]
Hi Dianna, I'll send a link shortly for you to fill out.

**Comment by Jegelewicz** [comment]
Actually - it would be best if you guys create all of the collections you are going to need at once - it is just more efficient.

**Comment by diannakrejsa** [comment]
**ASNHC:Bird
ASNHC:Herp
ASNHC:Tiss** 
Great, the three above are what we need for now! We're still operating under the cataloging system that only vouchered specimens get catalog numbers within their taxon collection (a tissue only mammal specimen does not get cataloged). So we'll need an ASNHC:Tiss if that is proper naming convention.

I have a feeling we'll need an **ASNHC:Para** as well since we're saving them back and have started organizing them into boxes and racks, but we don't have any data to migrate (just new, incoming data). Does it make better sense to make it or wait on it?

**Comment by campmlc** [comment]
Make it!

**Comment by dustymc** [comment]
Yes, it's MUCH more efficient to create all collections at once.

>ASNHC:Tiss

Technically: sure, no problem, and if everything always gets cited properly and such it'll be fine.

Practically, from lots of really painful experience: any chance to do something else? Best case, you're going to spend a lot of time building code tables and trying to figure out which "12" some citation refers to. Worst case - ask @jldunnum about the DGR....

**Comment by diannakrejsa** [comment]
Perusing the other collections' databases, let's add **ASNHC:Teach** to the list as well, while we're at it.

What's the problem been with a separate Tiss collection? Getting loans and linked citations correct?

**Comment by dbloom** [comment]
Is there a reason that mammals cannot be added to this, as well? At the
moment, VertNet is serving herps, birds and mammals to GBIF, VertNet and
iDigBio from an IPT that really shouldn't be used for public resources.
We're doing this for several good, historic reasons, and we can continue if
necessary, but it would be excellent to have these collections on an IPT
that is actually meant for public consumption.

**Comment by Jegelewicz** [comment]
They already have a mammal portal.

**Comment by Jegelewicz** [comment]
> Practically, from lots of really painful experience: any chance to do something else? Best case, you're going to spend a lot of time building code tables and trying to figure out which "12" some citation refers to. Worst case - ask @jldunnum about the DGR....

These are tissues for which there is no associated voucher, so maybe not as much of an issue? But still at some point there probably will be a citation of ASNHC 12 with some GenBank accession that will make it challenging to determine if it was from ASNHC:Mamm or ASNHC:Tiss

Honestly, now is the opportunity to just put those tissues in the proper collection - think on this one before you decide!

**Comment by diannakrejsa** [comment]
Oh I see, the secondary part of the prefix may get dropped. It's up to curators here ultimately, but I'll bring up the issues. Any other downstream effects or issues to bring up?

**Comment by dbloom** [comment]
That is news to us and as far as I can tell, it will be news to GBIF and
iDigBio, too. GBIF is serving a dataset from 2016 (2 years out of date from
the VertNet IPT which more current and which is the one that iDigBio and
VertNet both use - and that one hasn't been updated for a couple years at
least). We have these datasets on VN because Marcy knew the resources
served from Angelo State to be inaccurate and incomplete and didn't want
the portals to use those datasets - something to do with SilverBiology -
even thought we did our best to make them usable (which is why VertNet is
listed as Technical contacts for those three vert resources).

If there is yet another mammal dataset, it would be good to know where it
is so that I can notify the portals. If you are referring to the  dataset
that is currently in GBIF (
https://www.gbif.org/dataset/f9772d3a-db34-4a33-baa0-7a016a970358) to which
you are referring, I have no evidence that those data are current or being
updated. It would be good to know these things so that we are, at the
very least, not duplicating records in GBIF or elsewhere. Would you share
with me the most current information so that I can set all of the portals
to the correct data source?

**Comment by dustymc** [comment]
IF everything works like it should, it won't be a problem. 

If there's actually a clear separation and nothing is actually co-cataloged, it might not be much of a problem.

If it's the more or less normal situation where things are arbitrary, either by design or practice, experience suggests it will find a way to make you work way too hard for low-quality data.

Big-picture, I think simpler administrative situations are just easier for everyone to deal with if the politics-or-whatever allows.

**Comment by diannakrejsa** [comment]
I'll bounce this issue off them and see what they think (Curator of Mammals Bob Dowler and Curator of Genomic Resources Loren Ammerman). 

David, the ASNHC is in the process of moving all collections but the herbarium from Specify into Arctos. The mammal collection was migrated first and is complete (new records will still be incoming, but all data that was on Specify is now on Arctos). What needs to be done to avoid redundant records on other platforms? We'd like to overwrite anything on GBIF, iDigBio, VertNet because any edits to the data were done within the Specify client locally and, as you mention, have not been updated in 2+ years in database sharing platforms. 

Also, to clarify for me, what is the method by which Arctos data is shared via VertNet? Arctos serves data for its users to VertNet/GBIF/iDigBio and updates monthly?

**Comment by diannakrejsa** [comment]
Can we upload tissue-only specimens with just a tissue number (ASK in our case) and no catalog number? I think I've seen some records like this in Arctos before.

**Comment by dustymc** [comment]
All cataloged items in Arctos have a catalog number.

I could use some clarifying myself, but I can address the technical bits.

* There's a tool to generate EML (mostly from the data in Manage Collection) here:<img width="401" alt="Screen Shot 2019-12-13 at 8 03 41 AM" src="https://user-images.githubusercontent.com/5720791/70813958-4857ff00-1d7f-11ea-8e58-b08f75808312.png">

* VertNet has access to a slice of the database.
* The IPT data cache in that slice is currently updated monthly.

**Comment by Jegelewicz** [comment]
@dbloom they haven't published to GBIF from Arctos yet - literally just migrating data now, but the portal exists so we don't need to create one in Arctros.

**Comment by Jegelewicz** [comment]
> What needs to be done to avoid redundant records on other platforms? We'd like to overwrite anything on GBIF, iDigBio, VertNet because any edits to the data were done within the Specify client locally and, as you mention, have not been updated in 2+ years in database sharing platforms.

See #170

**Comment by diannakrejsa** [comment]
We'll publish to GBIF after we finish adding to Mammals, there are about 3.5K records left to get in.

Do we have a link to produce the ASNHC:Birds and other databases? We can hold off on ASNHC:Tiss for now if that's preferred, I'll talk to Bob and Loren and finalize the decision after break.

**Comment by Jegelewicz** [comment]
Start by completing the form [here](https://arctos.database.museum/new_collection.cfm). You can probably copy some of the stuff from your mammal collection. After you submit, I'll fill in the separate collections info. I might ask a few questions, then we'll have the collections created.

Thanks!

**Comment by campmlc** [comment]
I've preliminarily given admin approval. Do you have your institution and
collection codes formalized to create the GUIDs?

Also, Dianna, regarding separate tissue collections:  as an example, MSB
DGR used to be a separate collection, but we re-integrated the tissue
collection with the respective divisions because of the difficulty of
linking and documenting usage in the form of tissue loans and publications
and data divergence from the actual specimen vouchers.  I find it much
easier to manage tissues as part of the voucher collections themselves,
even when the tissues are all I have. You would need the curators to allow
cataloging of tissues only, which we do here at MSB. If there is no
voucher, as from mark/recapture studies or zoo or endangered species
recovery programs, we catalog the tissues. This allows tissues to link into
Arctos taxonomy and makes them searchable online in Arctos and via
aggregators such as GBIF and GGBN. If there is a voucher in another
institution, we catalog the tissues here and create a "same individual as"
relationship to the voucher, which is easy to do in Arctos and easy, but
less data-rich, when the voucher is in an external institution.
Alternatively, if you did have a separate tissue collection, you could use
these relationships to link tissues with separately cataloged vouchers.

The drawback or difficulty of integrating tissues with vouchers includes
requiring that the tissue collection manager has data access to the other
collections, in order to manage and link specimen parts, loans, and
publications. It also means that the tissue collection does not have
separate accessions or loans - these are managed through the other
divisions. So it depends on how integrated your collections are and how
well you all get along :). The advantage is better data integration and
usage tracking, and avoiding divergence of parts, data, and usage info in
different collections, avoiding different catnums for the different parts
of the same specimen in GBIF, etc. On the other hand, Arctos can handle
this complexity if you choose to go that route.

**Comment by diannakrejsa** [comment]
We'd like to use:
ASNHC:Bird
ASNHC:Herp
ASNHC:Para

We can hold off on ASNHC:Tiss for now. Personally, I have no problem cataloging tissue-only specimens or tissues where the voucher is held elsewhere. It is just against the tradition of having a present voucher for every cataloged specimen which has been the rule here. All divisions of the ASNHC get along quite well (there are six people including myself and curators), and coordinating with the other institutions where vouchers may be housed would probably be relatively simple. We'll talk about it more and see what we will do.

**Comment by campmlc** [comment]
OK, I'll need a short collection description for each. I'm assuming web url
and licensing etc are the same as your other collections?

**Comment by diannakrejsa** [comment]
Yep, should be same web url and same licensing.

ASNHC:Bird
The ASNHC Bird Collection totals approximately 2,500 specimens. The primary geographic area of representation is the western Edwards Plateau of Texas. A minor collection from Guyana, South America, is also present. A significant portion (about 35 percent) of the collection is of teaching material (zoo or pet trade specimens). The collection was established in 1968 and remained relatively small for 20 years. Within the last decade, collection organization has formalized and the rate of specimen addition has accelerated.

ASNHC:Herp
The ASNHC Amphibian and Reptile Collection totals approximately 15,500 specimens. Areas of strong geographic representation include west-central to north-central Texas including a large series from the Concho Valley. Significant series from surveys of Devil's River State Natural Area and Monahans Sandhills State Park are also present. Significant collections from northern Mexico are also present. The collection was established in the late 1960s.

ASNHC:Para
The ASNHC Parasite Collection is just being established. Many are ectoparasites and endoparasites of mammals.

**Comment by campmlc** [comment]
can't find licensing - CC0?

**Comment by diannakrejsa** [comment]
No idea! Haha. What does this mean and where might I find it?

**Comment by Jegelewicz** [comment]
You guys are set to CC BY

![image](https://user-images.githubusercontent.com/5725767/71212857-6508a100-226f-11ea-9f97-2bfa79eb4353.png)

**Comment by campmlc** [comment]
Ah yes, I forgot that the info is in manage collection (and not in Portals
- shouldn't it be?).

Here is explanation of licenses:
https://creativecommons.org/licenses/
but see
https://creativecommons.org/share-your-work/public-domain/cc0/
and
https://knowledge.figshare.com/articles/item/what-is-the-most-appropriate-licence-for-my-data
specifically, for publicly stored research:

   - *CC0 (Metadata only & datasets)*

CC0 can be particularly important for the sharing of metadata only, data
and databases, since it otherwise may be unclear whether highly factual
data and databases are restricted by copyright or other rights. Databases
may contain facts that, in and of themselves, are not protected by
copyright law. CC0 is recommended for data and databases and is used by
hundreds of organisations. It is especially recommended for scientific
data. Although CC0 doesn’t legally require users of the data to cite the
source, it does not take away the moral responsibility to give attribution,
as is common in scientific research.

But then, Arctos contains images and media, so maybe:


   - *CC-BY (figures, media, posters, papers, filesets)*

By licensing your research outputs under CC-BY, figshare ensures that your
research is openly available, but requires that others should give you
credit, in the form of a citation, should they use or refer to the research
object. This license lets others distribute, remix, tweak, and build upon
your work, even commercially, as long as they credit you for the original
creation. This is the most accommodating of licenses offered. Recommended
for maximum dissemination and use of licensed materials.

**Comment by Jegelewicz** [comment]
> But then, Arctos contains images and media, so maybe: 
>    - *CC-BY (figures, media, posters, papers, filesets)*

Every media object has it's own license. So this isn't necessary.

**Comment by diannakrejsa** [comment]
Should we stay with CC-BY for consistency or switch it all to CC0 then?

**Comment by diannakrejsa** [comment]
I'm diving back into the issue of the ASNHC:Tiss this week, so I'm quote Mariel's text from above as I ask and clarify a few things.

> I find it much easier to manage tissues as part of the voucher collections themselves, even when the tissues are all I have. [Cataloging tissue only] allows tissues to link into Arctos taxonomy and makes them searchable online in Arctos and via aggregators such as GBIF and GGBN. 
Would a Tiss database not be subjected to the same taxonomic trees as Mamm, Bird, and Herp? Would any of the rules, tools, and functions be different?

Would GBIF or the other aggregators have a problem serving our data if, say, mammal records were published by way of two databases, ASNHC:Mamm (for the vouchers + tissues) and ASNHC:Tiss (for the tissue-onlys)?

> Alternatively, if you did have a separate tissue collection, you could use these relationships to link tissues with separately cataloged vouchers. 
To clarify, we only would use the separate tissue collection for records where we do not (and, seemingly, will never) possess the voucher. All tissues associated with vouchers are integrated with the voucher as a part in the ASNHC:Mamm, ASNHC:Bird, and ASNHC:Herp databases. If the voucher exists but is housed at another museum, then any tissue that we possessed from that record would go to ASNHC:Tiss (if we go that route).

**Comment by Jegelewicz** [comment]
> Would a Tiss database not be subjected to the same taxonomic trees as Mamm, Bird, and Herp? Would any of the rules, tools, and functions be different?

Yes and No, respectively.

> Would GBIF or the other aggregators have a problem serving our data if, say, mammal records were published by way of two databases, ASNHC:Mamm (for the vouchers + tissues) and ASNHC:Tiss (for the tissue-onlys)?

I see nothing that would cause problems there.

**Comment by Jegelewicz** [comment]
> To clarify, we only would use the separate tissue collection for records where we do not (and, seemingly, will never) possess the voucher. All tissues associated with vouchers are integrated with the voucher as a part in the ASNHC:Mamm, ASNHC:Bird, and ASNHC:Herp databases. If the voucher exists but is housed at another museum, then any tissue that we possessed from that record would go to ASNHC:Tiss (if we go that route).

To be honest, I don't see why you would want to do this. It's just one more set of data to manage when the tissues could just be cataloged into their appropriate collections. There is absolutely nothing wrong with an occurrence for which the only part is a tissue.

**Comment by dustymc** [comment]
@Jegelewicz is entirely correct - collections are wholly administrative in Arctos.

Also agreed with the "don't see why..." comments; this looks like creating arbitrary complexity for the sake of arbitrary complexity. Losing or gaining a "voucher" and doing something necessarily-arbitrary, trying to make users understand why they need to search two collections to find stuff, some frog-bits being curated in one way and others in another, etc. - even if the really painful stuff never comes to pass, I can't see how this does anything other than make things less accessible to everyone.

>voucher exists but is housed at another museum

There's been a string of collections being shut down and materials transferred to another institution lately. The collection holding related material would be an obvious choice for recipient. Bringing those into a two-collection system would require yet more arbitrary administrative decisions - either fill your "tissue" collection up with bones, or recatalog and make it more difficult to find material from publications.

Collections are still entirely administrative, and we can do this without problem if necessary!

**Comment by campmlc** [comment]
From a biological perspective and research perspective and citation
perspective and overall data quality and workload perspective, it makes
much more sense to integrate the tissue collection with the respective
voucher collections. That is why we switched to that system at MSB.
However, there are still a couple of reasons that would make it desirable
to have a separate tissue collection.
1) As genomic resources collection manager, I have to be allowed manage
collection access to all collections that archive tissues. I have to be
able to create and manage accessions and loans for these collections, and
be able to see all restricted data and part locations, and be able to make
changes on my own authority without having to request permission from a
given division. Providing such access to a cm who may or may not be under
the supervision of a given curator may be problematic at some institutions.
2) For things like environmental samples or other genomic resources that
don't correspond to voucher specimens, and that are accessioned and managed
entirely by the tissue collection.
We still technically have kept our separate tissue collection portal,
although it isn't public, and I may need to revive the use of it especially
for 2) above.

**Comment by diannakrejsa** [comment]
I've extended this information to the Genomic Resources and Mammal Curators, but folks are busy with the end of the semester coursework. I'm on board with cataloging tissue-onlys; in this case, it's simpler for me (I'm the sole collections manager for all the divisions) and makes more sense if the tissue-only designation changes in time (the voucher is found, or the voucher is transferred to us). We have quite a few valuable un-cataloged occurrences with tissues but the voucher has been lost at some point. Right now these occurrences aren't searchable and are unknown to even us for the most part (held in a separate, out of the way spreadsheet, or are recorded in a tissue book but not digitized in any way). There is some possibility that their voucher will turn up at some point, so I like the idea of only keeping taxonomic divisions between databases. I'm trying to get approval on this, but if the decision is for an ASNHC:Tiss database thanks for being supportive of that!

**Comment by diannakrejsa** [comment]
Another pot I'm stirring--the ASNHC:Para database. We have a good number of parasites collected from and linked to host vouchers, but they are not well identified. Order-level taxonomic designations is basically as far as it gets. Is this worth assigning a catalog number to? 
Additionally, all parasites have been collected more as a "lot" with all ectos from a host vialed together and all endos from a host together. Since ticks and fleas may be housed together and other taxonomically diverging groups, should I denote them as 'parts' of their host (e.g. "ectoparasites") and leave it at that for now, and populate ASNHC:Para proper with cataloged records as we have individually stored and identified parasites? Any loss to their searchability going that route?

**Comment by Jegelewicz** [comment]
> Order-level taxonomic designations is basically as far as it gets. Is this worth assigning a catalog number to?

Absolutely! Then maybe someone will find them and ask to ID them for you!!!

**Comment by dustymc** [comment]
I would absolutely without question catalog them separately if it was a possibility, no matter the mix.

If you catalog them as parts they'll LOOK citable to users, not-quite-best case you'll end up with ABC:Host:123 (which leads to a dead rat and 500 parasites....).

If you catalog them separately, even as a mixed lot, I think users will be much more likely to notice that something's weird and request a citable identifier (at which point you can split the lot - some insect collections do that by eg sending extra barcodes and an excel template out on loans).

I suppose you COULD (more or less) catalog the whole shebang as "mouse and endos and ectos" which woul provide some limited searchability, but something like

* Mus musculus
* Arachnida {unsorted ectoparasites}
* Animalia {unsorted endoparasites}

is much more discoverable, and of course it's even better if you can get it to Nematomorpha or something. (Those are "A {string}" taxa formula - http://handbook.arctosdb.org/documentation/bulkloader.html#taxonomy)

**Comment by Jegelewicz** [comment]
> Additionally, all parasites have been collected more as a "lot" with all ectos from a host vialed together and all endos from a host together. Since ticks and fleas may be housed together and other taxonomically diverging groups, should I denote them as 'parts' of their host (e.g. "ectoparasites") and leave it at that for now, and populate ASNHC:Para proper with cataloged records as we have individually stored and identified parasites? Any loss to their searchability going that route?

I am pretty sure this is what MSB does. I do think there is a loss in searchability though. A person has to know to search for part name = endoparasite in a mammal or bird collection, which does not seem intuitive to me at all. I would vote for cataloging them as  - oh wait - what @dustymc said.

**Comment by diannakrejsa** [comment]
Gotcha! I guess being at MSB is where I got the idea, I was trying to figure out if that's best practices or there were reasons they went that way or wished they hadn't gone that way.

So, if and when a cataloged but mixed lot of parasites gets separated out, they maintain a shared catalog number but just get a unique barcode as they become individual parts?

**Comment by dustymc** [comment]
>they maintain a shared catalog number but just get a unique barcode as they become individual parts?

Fish do that, and it's arguably better than not doing that - a dedicated-enough collections manager who's familiar with the situation might find the relevant individual with enough work if the tags haven't fallen off or been duplicated or changed or ....

My advice is always "cite the item of scientific interest." I think that's the biological individual most of the time. Users can't cite IDs that don't exist. GUIDs are designed to be citable. I think the "proper" way to address all that is to recatalog the individual(s) as they're examined, and optionally use http://arctos.database.museum/info/ctDocumentation.cfm?table=CTID_REFERENCES&field=same%20lot%20as to link them back to the "source" lot. (We should probably have better guidance on when it's important to use that.)

**Comment by Jegelewicz** [comment]
> reasons they went that way or wished they hadn't gone that way.

As I understand it, they do that for curatorial reasons - the mammal people want to know what they collected. Eventually the stuff is transferred to the parasite people, who then catalog them and relate them back to the mammal record. If this happens fairly quickly, it isn't really a problem. If those "bulk" parts remain in the mammal record for years (or forever), they may get lost to parasitologists.

**Comment by Jegelewicz** [comment]
> We should probably have better guidance on when it's important to use that.

https://github.com/ArctosDB/arctos/issues/2239

**Comment by campmlc** [comment]
At MSB we do both.
Parasites are initially parts of the host record. Right now, ectoparasites
are only a single part name (no differentiation taxonomically).
Endoparasites can be broken down by cestode, nematode, trematode etc part
types.
This is what is recorded in the field, on host and parasite collection.
We are still developing a process whereby the parts get ID'd and cataloged
in MSB Para. Ideally, we'd wait until we had the opportunity to loan out
the lot of unsorted material, get it back with some sort of ID, and
catalog. We do that.
But now we have a backlog. So we are in the process of cataloging just as
"ectoparasites" which will go under "Arthropoda" as MSB:Para records. Then
as the lots get sent out, sorted, and Id'd, we would clone the catalog
record and assign new MSB:Para numbers as needed.
And yes, higher taxonomy is fine. It also is OK to create an ID of
"unknown". At least you know there was some sort of parasite! And the
records should be linked with shared identifiers (e.g. an NK or field
number, common to both), and shared colllecting event IDs.

**Comment by dustymc** [comment]
> It also is OK to create an ID of "unknown".

True, but it's useful to think of this at the scope of Arctos - "unknown" implies it could have been a motorcycle, sauropod, giant kelp, virus, .... Even something like "Animalia" adds a tremendous amount of precision.

>linked with shared identifiers (e.g. an NK or field number, common to both),

I certainly don't have a great way around that (2-part barcodes seem to do it but aren't always practical), but know that it WILL find a way to cause problems - the printer will get stuck and you'll end up with "1" printed on 2 mice and 12 tubes of parasites, etc., etc. Getting these cataloged under true unique identifiers and explicitly related to each other as quickly as possible is probably the best way to mitigate the inevitable problems caused by people writing numbers on things.

**Comment by campmlc** [comment]
Absolutely agree with dustymc; once they are cataloged, you have a way to
associate unique identifiers (guids, urls). Better to create the linkages
sooner rather than later while collectors etc are still around to verify
and catch any mistakes due to misread or mistyped field identifiers.
